frontend/src/lib/constants.tsx

```diff
@@ -11,6 +11,7 @@ export const ACTIONS_BAR_CHART = 'ActionsBar'
 export const ACTIONS_BAR_CHART_VALUE = 'ActionsBarValue'
 export const PATHS_VIZ = 'PathsViz'
 export const FUNNEL_VIZ = 'FunnelViz'
+export const FUNNELS_HISTOGRAM = 'FunnelsHistogram'
```

Comment:
Nit: In case we later change the type of visualisation, something like `FunnelTimeViz` might work better?